src/amo/components/SearchResult/index.js

```diff
@@ -123,6 +139,13 @@ export class SearchResultBase extends React.Component<InternalProps> {
         <div className="SearchResult-contents">
           <h2 className="SearchResult-name">
             {addon ? addon.name : <LoadingText />}
+            {showRecommendedBadge &&
+            _config.get('enableFeatureRecommendedBadges') &&
+            addon &&
+            addon.is_recommended &&
+            clientApp !== CLIENT_APP_ANDROID ? (
```

Comment:
Is there an open bug to support Android?

Reply:
I don't think so. This is more of a product issue than a technical issue. @jvillalobos and/or @devaneymoz do you know what the plans are regarding recommended add-ons on Android?

Reply:
We decided not to show the badge until we have a good recommendation process for Android. It's not a priority at the moment.